cmake/ctest/drivers/atdm/mutrino/local-driver.sh

```diff
@@ -10,6 +10,11 @@ source $WORKSPACE/Trilinos/cmake/ctest/drivers/atdm/ctest-s-driver-config-build.
 
 set -x
 
-/usr/local/bin/salloc -N 1 -p standard -J $JOB_NAME \
-  --time=${SALLOC_CTEST_TIME_LIMIT_MINUTES} \
-  $WORKSPACE/Trilinos/cmake/ctest/drivers/atdm/ctest-s-driver-test.sh
+atdm_run_script_on_compute_node \
+  $WORKSPACE/Trilinos/cmake/ctest/drivers/atdm/ctest-s-driver-test.sh \
+  $PWD/ctest-s-driver-test.out \
+  ${SALLOC_CTEST_TIME_LIMIT_MINUTES}
+
+#/usr/local/bin/salloc -N 1 -p standard -J $JOB_NAME \
```

Comment:
@fryeguy52, we should remove this commented-out command in a follow-up commit and PR (I don't want to holdup the merge of this PR).  Git can show this change better than a comment like this.